05.ls/file_sortable.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module FileSortable
+  def sorted_files(files_or_path, opts)
+    files = if files_or_path.instance_of?(String)
+              Dir.entries(files_or_path)
+            else
+              files_or_path
+            end
+
+    files.sort!
+
+    sorted_files = if opts[:a]
+                     files
+                   else
+                     files.reject { |file| file.start_with?('.') unless file.include?('/') }
```

Comment:
```suggestion
                     files.reject { |file| File.basename(file).start_with?('.') }
```

でもいいのかな？と思ったんですがどうでしょう。

https://docs.ruby-lang.org/ja/latest/method/File/s/basename.html

Reply:
ありがとうございます。修正しました。